.gitignore

```diff
@@ -22,6 +22,7 @@ coverage
 yalc.lock
 /test-results.xml
 package-lock.json
+src/applications/proxy-rewrite/tests/__image_snapshots__/__diff_output__/
```

Comment:
I would like to consolidate our VRT configurations so that they all output to a single directory at the root of the project so we don't have to have such a deep gitignore config.